macros/parserDifferenceQuotient.pl

```diff
@@ -15,28 +15,27 @@
 ################################################################################
 
 =head1 NAME
-
 parserDifferenceQuotient.pl - An answer checker for difference quotients.
-
 =head1 DESCRIPTION
-
```

Comment:
I'm going to close this pull request, make the changes, and submit another pull request.